### ISSUE TITLE
Scaletesting: POC a testing pipeline

### ISSUE THREAD

**Issue by jhchabran** [sourcegraph/sourcegraph#43042]
In the future, we'll want to be able to run automated scale testing scenarios and to gather some sort of report to tell us if it went okay or not. 

So to make a first step in that direction, we need the following: 

- [ ] Have a place to locate _scripts_ (in the loose sense, we don't care which form they take) that are driving the tests. 
- [ ] Be able to target any Sourcegraph instance. 
- [ ] Run in a separate CI pipeline. 
- [ ] Gather artefacts needed to understand what went wrong. 

Scenario to use as a reference: https://github.com/sourcegraph/sourcegraph/issues/43043 (Please note that it's totally fine to dumb it down to something much simpler while automating it).

**Comment by sanderginn** [comment]
Closing, not in scope with current Job Fair projects.

**Comment by jhchabran** [comment]
To anyone looking to take over this as part of their work in a job fair project: this is very similar that what we're doing in the e2e tests. Ping us on #dev-experience if you need more pointers.
